openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -28,29 +28,31 @@
 - control_key: AU-2
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-155
+        OpenShift has an API audit system which would need to be enabled
+        and configured for the organizational requirmennts.
     - key: b
       text: |
         Coordinating the security audit function with other
         organizational entities is outside the scope of
         system configuration.
     - key: c
       text: |
+        Coordinating the security audit function with other
+        organizational entities is outside the scope of
+        system configuration.
+
         For rationale regarding why the auditable events are deemed to be
         adequate to support after-the-fact investigations of security incidents,
         please contact your countries Common Criteria representative:
         https://www.commoncriteriaportal.org/ccra/members/
```

Comment:
Coordination is part of AU-2(b), but not AU-2(c). Reference NIST language:

``b. Coordinates the security audit function with other organizational entities requiring audit-related information to enhance mutual support and to help guide the selection of auditable events;

c. Provides a rationale for why the auditable events are deemed to be adequate to support after-the-fact investigations of security incidents; and
``